app/src/main/java/no/usn/mob3000/ui/screens/user/ProfileEditScreen.kt

```diff
@@ -73,7 +73,8 @@ fun ProfileEditScreen(
                 value = avatarUrl,
                 onValueChange = { avatarUrl = it },
                 label = { Text(stringResource(R.string.profile_edit_avatar_url)) },
-                placeholder = { Text(stringResource(R.string.profile_edit_avatar_url_placeholder)) },
+                placeholder = { Text(stringResource(R.string.profile_edit_avatar_url_placeholder), color = MaterialTheme.colorScheme.onBackground.copy(alpha = 0.6f)) },
```

Comment:
Same as [with this comment](https://github.com/frigvid/mob3000-group11/pull/27/files#r1847357323). Since there are multiple parameters, it should be indented.